### DISCUSSION TITLE
Bulk Actions

### DISCUSSION THREAD

**Discussion by saloeric** [rappasoft/laravel-livewire-tables#1225] [General]
Hello, I'm experiencing a small bug. When I select all records by clicking on the multi-select checkbox, it shows that everything is selected and provides a button to "Deselect All". However, when I click on this button, the entire table is deselected, but the multi-select checkbox remains active.

I am using the latest version of this package (Bootstrap 4).

Thank you.

**Comment by lrljoe** [comment]
Thanks, can you confirm you're on the latest version of the package?

When you say it remains active, do you mean it stays checked?

**Comment by saloeric** [comment]
Hello, version 2.14. Yes, it remains selected.

**Reply by lrljoe** [reply]
Thanks, I'll take a look and get a fix in for next release
